### PR TITLE
Move valid inputs from reorder_philips into module gradients

### PR DESCRIPTION
**Pull request by arnaudbore**

### CODE REVIEW

**Comment by arnaudbore** [comment]
Build passed ! Good Job :beers: !

File: scilpy/gradients/utils.py [hunk 1]
```diff
@@ -29,3 +29,57 @@ def random_uniform_on_sphere(nb_vectors):
     bvecs[:, 2] = np.cos(theta)
 
     return bvecs
+
+
+def get_new_order_philips(philips_table, dwi, bvals, bvecs):
+    """
+    Reorder bval and bvec files
```
Comment (EmmaRenauld) [inline scilpy/gradients/utils.py:36]: Can we be more precise? Based on ?

**Review by EmmaRenauld — APPROVED** [review]

**Comment by arnaudbore** [comment]
Build Failed :boom: 
https://jenkins.scil.usherbrooke.ca/job/scilpy/job/PR-787/2/

**Comment by arnaudbore** [comment]
Build Failed :boom: 
https://jenkins.scil.usherbrooke.ca/job/scilpy/job/PR-787/3/

**Comment by arnaudbore** [comment]
Build passed ! Good Job :beers: !

**Pull request merged**